### PR TITLE
Docs: Material change in pom, "or" is a "non-exclusive"

### PR DESCRIPTION
**Pull request by lokesh090297**
Updated the POM documentation

### CODE REVIEW

File: content/apt/guides/introduction/introduction-to-the-pom.apt [hunk 1]
```diff
@@ -212,8 +212,8 @@ Introduction to the POM
  parent of our POM. And we do so by specifying the fully qualified artifact name of the parent POM. With this setup, our
  module can now inherit some of the properties of our parent POM.
 
- Alternatively, if we want the groupId and / or the version of your modules to be the same as their parents, you can
- remove the groupId and / or the version identity of your module in its POM.
+ Alternatively, if we want the groupId and the version of your modules to be the same as their parents, you can
```
Comment (elharo) [inline content/apt/guides/introduction/introduction-to-the-pom.apt:215]: we --> you

File: content/apt/guides/introduction/introduction-to-the-pom.apt [hunk 2]
```diff
@@ -229,7 +229,7 @@ Introduction to the POM
 </project>
 +-----+
 
- This allows the module to inherit the groupId and / or the version of its parent POM.
+ This allows the module to inherit the groupId and the version of its parent POM.
```
Comment (elharo) [inline content/apt/guides/introduction/introduction-to-the-pom.apt:232]: Is it possible someone wants to inherit the group ID but not the artifact ID or version?

**Review by elharo — COMMENTED** [review]
This is not a typo. The text might be incorrect as written — I'm not sure — but this is a material change. Please update the PR description.

File: content/apt/guides/introduction/introduction-to-the-pom.apt (hunk 2 above)
Comment (lokesh090297) [reply content/apt/guides/introduction/introduction-to-the-pom.apt:232]: yes, it's possible!

**Comment by lokesh090297** [comment]
@elharo 
"or" is a "non-exclusive or" which means "either A or B **or** A and B".

**Review by elharo — APPROVED** [review]

**Comment by lokesh090297** [comment]
@elharo can you merge this PR?
 OR please tag someone who can merge it.

**Review by elharo — COMMENTED** [review]
checking jenkins: https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-site/job/or/

**Comment by lokesh090297** [comment]
https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-site/job/or/ 
@elharo Everything is fine in the build.
Thanks!

**Pull request merged**